x-pack/plugins/ingest_manager/server/services/saved_object.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function escapeSearchQueryPhrase(val: string) {
```

Comment:
can you add a comment and an example of input and output of a malformed value. In addition can you add the return type of the function.